data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,436 @@
+			`	"You may act as the oversight for the human governments in this project. Proposal D-86513-7P4 is approved. Council adjourned."`
+			`	When you meet Sayari afterwards a touch of her usual enthusiasm returns as you share the news.`
+			`	"This is fantastic news, now all that remains to be done is for you to escort the construction fleet from Makerplace to Ultima Thule and observe the initial stages. They'll be waiting for your arrival on Makerplace."`
+				accept
+	on accept
+		event "role-play station before construction"
+		log `The diplomats have elected to double-down on the role-play community angle, citing its success. However, the Hai are unwilling to trust a human government to see the inner workings of a station under construction in the wake of the civil war, and the Republic will not allow a station to be built on their side unsupervised. You have been asked to fulfill the role of supervisor as a result. The Elders asked you some odd questions, but agreed that you were an acceptable individual for the task.`
+	on stopover
+		dialog `A fleet of seven Geocoris are waiting to accompany you when you arrive, with just the final items being loaded as you come into land. They will follow you to the site where the new station will be built.`
+	on complete
+		event "role-play station after construction"
+	
+
+mission "Hai Building Mountaintop [1a] Construction Escorts"
+	landing
+	invisible
+	source "Makerplace"
+	destination "Mountaintop"
+	clearance
+	to offer
+		has "Hai Building Mountaintop [1] Construction Fleet: active"
+	on visit
+		dialog `You have arrived without the seven Geocoris you are escorting from <origin> to <destination>. You must take off and wait for them to finish this mission.`
+	npc accompany save
+		personality escort timid
+		government "Hai (Wormhole Access)"
+		ship Geocoris "Du Cor Hey"
+		ship Geocoris "Mon Yo Sen"
+		ship Geocoris "Vai e Lur"
+		ship Geocoris "Du Woa Ow"
+		ship Geocoris "Luf Sook Hey"
+		ship Geocoris "Du Eh"
+		ship Geocoris "Mon e Ow"
+		ship Geocoris "Eek lo Pok"
+
+mission "Hai Building Mountaintop [1b] Return"
+	landing
+	name "Return to <planet>"
+	description `The Hai have another job for you at <destination>.`
+	source "Mountaintop"
+	destination "Hai-home"
+	clearance
+	to offer
+		has "Hai Building Mountaintop [1a] Construction Escorts: done"
+	on offer
+		conversation
+			`On arrival, you see one of the Geocoris deploy a steel ring around itself. Space-based cranes set about removing the three outer sections of the Geocoris and moving them to the steel ring. After they finish with the first, they progressively dismantle the other seven Geocoris you escorted. Within a few hours, there is a makeshift station with eight Geocoris power generators and a sizable amount of storage and habitat space. The construction workers begin unpacking the contents of the Geocoris storage pods, checking construction seals, engaging principle life support systems, and a massive holoprojector.`
```

Comment:
So, the use of it as plural was only un-deprecated on wiktionary *after* this was originally written. There remains very few instances of it and there are also 0 results on google's ngram viewer for the use of Geocorises, so we'd decided that the plural of Geocoris was Geocoris. Particularly because the plural spp. referenced in applicable texts appear to use Geocoris. (ala `2. Geocoris spp [plural species].`)

Reply:
```suggestion
			`On arrival, you see one of the Geocoris deploy a steel ring around itself. Space-based cranes set about removing the three outer sections of the Geocoris and moving them to the ring. After they finish with the first, they progressively dismantle the other seven Geocoris you escorted. Within a few hours, there is a makeshift station with eight Geocoris power generators and a sizable amount of storage and habitat space. The construction workers begin unpacking the contents of the Geocoris storage pods, checking construction seals, engaging principle life support systems, and erecting a massive holoprojector.`
```
Just adding in @Bardagh 's suggestion and one fix: you need a verb for `holoprojector`, or you could rephrase it so it's grammatically correct. I'd suggest plopping in a verb b/c I think it'd sound smoother than the rephrases I can think of.